hello-world/AKS/README.md

```diff
@@ -10,8 +10,6 @@ The container is hosted publicly on [Azure Container Registry](fishersnpregistry
 
 ## Getting Started
 
-To build the container run `./get_verbose_report.sh` from the `hello-world` root directory. This also copies the verbose-report executable for the ACI example.
-
 Then run `docker build -t <image-name:tag> .` from the same directory as the Dockerfile.

```

Comment:
I guess "then" can be removed? The reader might think that there's a missing step.

Reply:
Good catch.